website/pages/docs/theming/scale-out-theme.mdx

```diff
@@ -0,0 +1,519 @@
+You can write the Global Styles override here.
+
+```ts live=false
+// styles.ts
+
+import { Styles } from '@chakra-ui/theme-tools'
+
+const styles: Styles = {
+  global: () => ({
+    body: {
+      // These variables are defined in /foundations
+      fontWeight: 'normal',
+      color: 'black',
+      bgColor: 'white',
+      fontSizes: '16px',
+      lineHeight: 'normal',
+      // More...
+    },
+  }),
+}
+
+export default styles
+```
+
+---
+
+## 📁 Foundations
+
+<ComponentLinks
+  github={{ package: "theme/src/foundations/" }}
+/>
+
+In this folder, we define the style variables that are commonly used in all components,
+such as `colors`, `spacing`, `shadows`, and `typography`.
+All of these settings will be combined with the default theme settings.
+
+### 📐 Borders
+You can add additional borders like this:
+
+```ts live=false
+// borders.ts
+
+const borders = {
+  none: 0,
+  "16px": "16px solid",
+  "dashed": "2px dashed",
+  "double": "2px double",
+  // More...
+}
+
+export default borders
+```
+
+### 🚩 BreakPoints 
+You can overwrite the default breakpoints like this:
+
+```ts live=false
+// breakpoints.ts
+
+import { createBreakpoints } from '@chakra-ui/theme-tools'
+
+// Breakpoints for responsive design
+const breakpoints = createBreakpoints({
+  sm: '768px',
+  md: '1024px',
+  lg: '1216px',
+  xl: '1408px',
+})
+
+export default breakpoints
+
+```
+To learn more about usage of breakpoints, [see "Responsive Styles" Page](/docs/features/responsive-styles).
+
+
+###  🎨 Colors
+If you want to use additional colors, this setting is useful.
+You can also see [default colors list here](/docs/theming/theme#colors).
+
+```ts live=false
+// colors.ts
+
+export interface ColorHues {
+  50: string
+  100: string
+  200: string
+  300: string
+  400: string
+  500: string
+  600: string
+  700: string
+  800: string
+  900: string
+}
+
+export type Colors = typeof colors
+
+const colors = {
+  youtube: "#FF0000",
+  pinkAlpha: {
+    50: "rgba(252, 157, 157, 0.04)",
+    100: "rgba(252, 157, 157, 0.06)",
+    200: "rgba(252, 157, 157, 0.08)",
+    300: "rgba(252, 157, 157, 0.16)",
+    400: "rgba(252, 157, 157, 0.24)",
+    500: "rgba(252, 157, 157, 0.36)",
+    600: "rgba(252, 157, 157, 0.48)",
+    700: "rgba(252, 157, 157, 0.64)",
+    800: "rgba(252, 157, 157, 0.80)",
+    900: "rgba(252, 157, 157, 0.92)",
+  },
+  // More...
+}
+
+export default colors
+```
+
+### 🌈 Border Radius
+You can customize border radius like this:
+
+```ts live=false
+// radius.ts
+
+const radii = {
+  none: "0",
+  base: "7px",
+  "circle": "50%",
+  // More...
+}
+
+export type Radii = typeof radii
+
+export default radii
+```
+
+### 🌓 Shadows
+You can add additional box-shadows like this:
+
+```ts live=false
+// shadows.ts
+
+const shadows = {
+  shopify: 'rgba(0, 0, 0, 0.15) 0px 5px 15px 0px',
+  airbnb: 'rgba(0, 0, 0, 0.15) 0px 2px 8px',
+  mac: 'rgba(0, 0, 0, 0.2) 0px 20px 30px',
+  // More...
+};
+
+export type Shadows = typeof shadows
```

Comment:
Please remove any additional TypeScript type when it is not clear for what it should be used.